### ISSUE TITLE
SNIMissingWarning and InsecurePlatformWarning on Ubuntu LTS

### ISSUE THREAD

**Issue by bernikr** [rahiel/telegram-send#5]
Im using `Bash on Ubuntu on Windows`, which is more or less equvalent to `Ubuntu 14.04.5 LTS` and has `python 2.7.6` installed.
When using `telegram-send` or the Telegram backbone of ntfy it works, but I get the following warning:

```
/usr/local/lib/python2.7/dist-packages/urllib3/util/ssl_.py:334: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  SNIMissingWarning
/usr/local/lib/python2.7/dist-packages/urllib3/util/ssl_.py:132: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
```

**Comment by rahiel** [comment]
This is due to a bug in Python itself, this was fixed in version 2.7.10 so you'll need that one or newer.

In your case I think you can switch to Python 3, uninstall ntfy and telegram-send: `pip uninstall ntfy telegram-send`, and then install it with Python 3, using `pip3 install ntfy[telegram]`.

**Comment by bernikr** [comment]
Thanks, that worked. Although I had to install ntfy and telegram-send seperatly as `pip3 install ntfy[telegram]` presented me with an error...
